models/team_test.go

```diff
@@ -95,3 +158,11 @@ func checkTeamInvertedIndex(t *testing.T, c aetest.Context, got *Team, want test
 
 	return errors.New("team not found")
 }
+
+// errString returns the string representation of an error.
+func errString(err error) string {
```

Comment:
yes I saw this, I will refactor them in another issue, maybe even create a testing package :)